src/azure-cli/azure/cli/command_modules/synapse/constant.py

```diff
@@ -62,3 +64,17 @@ class SqlPoolConnectionClientAuthenticationType(str, Enum):
     SqlPassword = 'SqlPassword'
     ActiveDirectoryPassword = 'ADPassword'
     ActiveDirectoryIntegrated = 'ADIntegrated'
+
+
+class PrincipalType(str, Enum):
+    user = "User"
+    group = "Group"
+    service_principal = "ServicePrincipal"
+
+
+class ItemType(str, Enum):
+    bigDataPools = "bigDataPools"
+    scopePools = "scopePools"
```

Comment:
Why `scopePools` here since it has been removed from `ITEM_NAME_MAPPING`? Should it be removed too?

Reply:
Thanks for catching that, will remove it.